packages/webamp/js/reducers/tracks.ts

```diff
@@ -6,6 +6,27 @@ export interface TracksState {
   [id: string]: PlaylistTrack;
 }
 
+let finalKhz: String = 'NaN';
+let finalKbps: String = 'NaN';
+function massageKhz(khz: number) {
```

Comment:
Maybe we could get a comment with a link to the source of truth we are emulating here?